auth/src/main/java/org/jboss/pnc/auth/ExternalAuthentication.java

```diff
@@ -31,45 +32,33 @@
 import org.apache.http.client.methods.HttpPost;
 import org.apache.http.message.BasicNameValuePair;
 import org.jboss.logging.Logger;
+import org.jboss.pnc.common.json.moduleconfig.AuthenticationModuleConfig;
 import org.keycloak.OAuth2Constants;
 import org.keycloak.RSATokenVerifier;
-import org.keycloak.VerificationException;
-import org.keycloak.adapters.AuthChallenge;
 import org.keycloak.adapters.HttpClientBuilder;
-import org.keycloak.adapters.KeycloakDeployment;
-import org.keycloak.adapters.installed.KeycloakInstalled;
+import org.keycloak.common.VerificationException;
+import org.keycloak.common.util.KeycloakUriBuilder;
 import org.keycloak.constants.ServiceUrlConstants;
 import org.keycloak.representations.AccessToken;
 import org.keycloak.representations.AccessTokenResponse;
 import org.keycloak.util.JsonSerialization;
-import org.keycloak.util.KeycloakUriBuilder;
 
 public class ExternalAuthentication {
```

Comment:
This one could be moved to integration-tests because we still need some piece of code to authenticate user and token in the tests.